### ISSUE TITLE
ACE editor disables default browser spell-checking  

### ISSUE THREAD

**Issue by monperrus** [gnieh/bluelatex#198]
ACE editor disables default browser spell-checking that is available in textarea.

How to restore spell-checking? Should bluelatex embed its own spell-checker library?

**Comment by monperrus** [comment]
up for this issue. spell-checking is really important.

**Comment by tdurieux** [comment]
With ace editor it's not possible to use the browser spellcheck. 
I have to create a spell check, the client needs to download the dictionary (+- 1mo) and add several dependencies. (example of spell check ace extension: https://github.com/zedapp/spellchecker or https://github.com/benweet/stackedit/blob/dev/public/res/extensions/spellCheck.js)

It's not a small change.

**Comment by monperrus** [comment]
> With ace editor it's not possible to use the browser spellcheck. 
are you 100% sure of this?

--Martin

**Comment by tdurieux** [comment]
Yes, see discussions ace issues (ex: https://github.com/ajaxorg/ace/issues/2107)

**Comment by monperrus** [comment]
I see.

> It's not a small change.
>
discussion to be continued then

--Martin
